specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/DatabaseExtensions.json

```diff
@@ -277,6 +354,45 @@
           "description": "Error message.",
           "type": "string",
           "readOnly": true
+        },
+        "queuedTime": {
+          "description": "Queued time.",
+          "type": "string",
```

Comment:
is it e.g. "format": "date-time"?

https://swagger.io/docs/specification/data-models/data-types/#:~:text=OpenAPI%20defines%20the%20following%20built-in%20string%20formats%3A%20date,by%20RFC%203339%2C%20section%205.6%2C%20for%20example%2C%202017-07-21T17%3A32%3A28Z

Reply:
this is intended for sql usage.

Reply:
The format for the date would be like: "queuedTime": "2/2/2020 8:33:27 PM"
Does this answer your question?

Reply:
Logic is same as the one for [ImportDatabase.json](https://msdata.visualstudio.com/Database%20Systems/_git/DsMainDev?path=%2FSql%2Fxdb%2Fmanifest%2Fsvc%2Fmgmt%2FWebApiHosting%2FARM%2FAutoGenSwagger%2FMicrosoft.Sql%2Fpreview%2F2020-02-02-preview%2Fexamples%2FImportDatabase.json&_a=contents&version=GBmaster)

Reply:
I'm suggesting you can either add a 'format' property in the swagger here to give some hinting to swagger consumers, code generators etc, as to the actual format, or at least mention it in the description, so people don't have to guess.

Reply:
fix in DSMain

Reply:
we might fix it in 2021-08-01

Reply:
Yes, we can fix it in 2021-08-01 if that's ok with you.